mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -26,9 +26,13 @@
 	</div>
 
 	<div class="task-container">
-		<aui:input cssClass="input-item" name="task" placeholder="Enter your task.." />
+		<aui:form>
+			<aui:input cssClass="input-item" name="task">
+				<aui:validator errorMessage="Please enter some text." name="required" />
```

Comment:
Use localization for all labels in UI. Do not use plain text, such as "Please enter some text.".

See docs https://dev.liferay.com/develop/tutorials/-/knowledge_base/6-2/localization
See examples of implementation in [Music Portlet](https://github.com/markocikos/liferay-internship/tree/master/matea-pjanic/assignments/liferay-app/sdk/portlets/music-portlet) or [Perzej Portlet](https://github.com/markocikos/liferay-internship/tree/master/kresimir-coko/assignments/liferay-app/sdk/portlets/perzej-portlet)